docs/examples/objc/iOS-Carthage/ThemisTest.xcodeproj/project.pbxproj

```diff
@@ -247,6 +254,7 @@
 				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
 				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = NO;
```

Comment:
This looks cool. I remember `#include "themis/things.h"` biting us with completely unrelated error messages. It's nice if that's a proper warning now.